x-pack/test/functional/apps/dashboard/migration_smoke_tests/lens_migration_smoke_test.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* This test is importing saved objects from 7.13.0 to 8.0 and the backported version
+ * will import from 6.8.x to 7.x.x
+ */
+
+import expect from '@kbn/expect';
+import path from 'path';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+  const testSubjects = getService('testSubjects');
+  const dashboardPanelActions = getService('dashboardPanelActions');
+
+  const PageObjects = getPageObjects(['common', 'settings', 'header', 'savedObjects', 'dashboard']);
+
+  describe('Export import saved objects between versions', () => {
+    before(async () => {
+      await esArchiver.loadIfNeeded(
+        'x-pack/test/functional/es_archives/getting_started/shakespeare'
+      );
+      await kibanaServer.uiSettings.replace({});
+      await PageObjects.settings.navigateTo();
+      await PageObjects.settings.clickKibanaSavedObjects();
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/getting_started/shakespeare');
+      await esArchiver.load('x-pack/test/functional/es_archives/empty_kibana');
+    });
+
+    it('should be able to import dashboard with various Lens panels from 7.12.1', async () => {
```

Comment:
Should this not happen as part of the `before` setup? I might be missing something though!

Reply:
It could be a part of the `before` setup because it is required by all subsequent tests - I moved it into its own test because the test fails if a migration error occurs, and I wanted the failed test name to be more explicit. I will take a look into moving most of this into `before` but leaving the import success validation as its own test.